rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -62,245 +49,62 @@ class Airdrop(NamedTuple):
-        'https://gist.githubusercontent.com/LefterisJP/1a293bf46b388f709df84ff98c5c5cc6/raw/196ccb50451d908d71cca4bc43731d6547b2276b/poap_medalla.json',
-        'https://poap.delivery/medalla',
-        'Medalla Validator',
-    ),
-    'muir_glacier': (
-        # is checksummed
-        'https://gist.githubusercontent.com/LefterisJP/d135745cf9f4f3143555e0f6a8f0d804/raw/d34c93087100168cac0dfd0ab46254b4a82ff0b8/poap_muir_glacier.json',
-        'https://poap.delivery/muir-glacier',
-        'Muir Glacier',
-    ),
-    'proof_of_gucci': (
-        # is checksummed
-        'https://gist.githubusercontent.com/LefterisJP/43b2c4bb73923d7bb3eaf3b329f7f7a1/raw/e5adb753a3a86f0a50bf93137e2c4adc61548293/poap_proof_of_gucci.json',
-        'https://poap.delivery/proof-of-gucci',
-        'YFI - Proof of Gucci',
-    ),
-    'proof_of_gucci_design_competition': (
-        # is checksummed
-        'https://gist.githubusercontent.com/LefterisJP/40c6b2a94d4b8d7f442522b099e5e258/raw/a0851ce92ca2f668e17a97c9df982bc3e12f9bb3/poap_proof_of_gucci_design_competition.json',
-        'https://poap.delivery/proof-of-gucci-design',
-        'YFI - Proof of Gucci Design Competition',
-    ),
-    'resuscitators': (
-        # is checksummed
-        'https://gist.githubusercontent.com/LefterisJP/0ac0216f82f16453b74a40529384a152/raw/f003083090efd834bcee1d0d1fe4e218380aa0cf/poap_resucitators.json',
-        'https://poap.delivery/medalla-resuscitator',
-        'Medalla Resuscitators',
-    ),
-    'yam': (
-        # is checksummed
-        'https://gist.githubusercontent.com/LefterisJP/d676e1f3db8df96928c2501c6be434ac/raw/fa0a603117f6e3e807a49491924aaaa2bc89179a/poap_yam.json',
-        'https://poap.delivery/yam',
-        'Yam Heroes',
-    ),
-    'ycover': (
-        # is checksummed
-        'https://gist.githubusercontent.com/LefterisJP/727c05f3a9cab79059258528595c102e/raw/59ff367bd532f632ce94e69900709155a55de82a/poap_ycover.json',
-        'https://poap.delivery/ycover',
-        'A New Face For yCover',
-    ),
-    'yfi_og': (
-        # is checksummed
-        'https://gist.githubusercontent.com/LefterisJP/58862ec2b398c770d60c24e50e18e50c/raw/3292995f530baedcdacde02526c6b2bd1de0e110/poap_yfi_og.json',
-        'https://poap.delivery/yfi-og',
-        'I Played 4 YFI',
-    ),
-}
+    May raise - RemoteError if the request fails or metadata is invalid.
+    """
+    try:
+        response = requests.get(url=AIRDROPS_INDEX, timeout=CachedSettings().get_timeout_tuple()).json()  # noqa: E501
+        airdrops: dict[str, Airdrop] = {}
+        for protocol_name, airdrop_data in response['airdrops'].items():
+            if 'new_asset_data' in airdrop_data:
+                if AssetType(airdrop_data['new_asset_data']['asset_type']) == AssetType.EVM_TOKEN:
+                    asset = airdrop_data['asset'] = get_or_create_evm_token(
+                        userdb=database,
+                        evm_address=airdrop_data['asset_identifier'].split(':')[-1],
```

Comment:
I would leave the address also in the data. Is not much and it prevents you from making mistakes here